exe/solr_wrapper

```diff
@@ -73,6 +93,17 @@ end
 options[:verbose] = true if options[:verbose].nil?
 
 instance = SolrWrapper.instance(options)
+
+if command == 'clean'
+  if instance.started?
+    $stderr.print "Please stop soll before cleaning\n"
```

Comment:
`s/soll/solr/`, please.

And, can we just make these `.puts` without the trailing `\n`?